lib/matplotlib/backends/qt_editor/_formlayout.py

```diff
@@ -444,15 +443,13 @@ def __init__(self, data, title="", comment="",
         # Button box
         self.bbox = bbox = QtWidgets.QDialogButtonBox(
             QtWidgets.QDialogButtonBox.StandardButton(
-                _to_int(
-                    _enum("QtWidgets.QDialogButtonBox.StandardButton").Ok) |
-                _to_int(
-                    _enum("QtWidgets.QDialogButtonBox.StandardButton").Cancel)
+                    QtWidgets.QDialogButtonBox.StandardButton.Ok |
+                    QtWidgets.QDialogButtonBox.StandardButton.Cancel
```

Comment:
I'm unsure why you can remove `_to_int` from here, but nowhere else?

Reply:
This may be a rebase artifact... I'll look into it.

Reply:
It may be that this was on purpose based on some version reasoning, but I've readded it to be on the safe side.